backoffice/www/handlers/user.py

```diff
@@ -0,0 +1,31 @@
+import tornado
+from tornado_sqlalchemy import SessionMixin
+
+from backoffice.model.user import User
+from backoffice.www.handlers.base import BaseBOHandler
+
+
+class CreateUserHandler(SessionMixin, BaseBOHandler):
+  ROUTE = '/create_user'
+
+  def get(self):
+    count = self.session.query(User).count()
+    self.write('{} users so far!'.format(count))
+
+
+class UserJson(SessionMixin, BaseBOHandler):
+  ROUTE = '/users_json'
+
+  @tornado.web.authenticated
+  def get(self):
+    users = self.session.query(User).all()
+    # self.write({"users": users},)
+    self.write({"data": User.serialize_list(users)})
+
+
+class ListUserHandler(SessionMixin, BaseBOHandler):
+  ROUTE = '/list_users'
+
+  @tornado.web.authenticated
+  def get(self):
+    self.render("list_users.html")
```

Comment:
I feel like this is missing the data? It's just invoking the template but we should pass the list of users that currently exist, like it's done in the /users_json handler (which I believe it's just for debug, which is great to have)